tests/page_test.go

```diff
@@ -399,7 +399,7 @@ func TestPageScreenshotFullpage(t *testing.T) {
 
 		const div = document.createElement('div');
 		div.style.width = '1280px';
-		div.style.height = '8000px';
+		div.style.height = '800px';
```

Comment:
Can we not lower these resolutions even further? what's stopping us from using 800x600? Or even 1x1?

Reply:
I get a error on the last assert when i change it 800x600 🤷